bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Trajectory/Orbit.cpp

```diff
@@ -375,6 +375,54 @@ inline void OpenSpaceToolkitAstrodynamicsPy_Trajectory_Orbit(pybind11::module& a
                 )doc"
             )
 
+            .def_static(
+                "frozen",
+                &Orbit::Frozen,
+                arg("epoch"),
+                arg("altitude"),
```

Comment:
Shouldn't we provide SMA instead of altitude as the argument here since that is more standard?
Is it geodetic altitude, geocentric altitude, etc? Or at least specify that it is geocentric altitude in this case